setup.py

```diff
@@ -30,14 +30,18 @@
             "aiohttp==3.7.4.post0",
             "slackclient==2.7.2",
             "responses==0.10.14",
-            "google-cloud-translate==2.0.1",
             "paramiko>=2.8.0",
+            "google>=3.0.0",
+            "google-cloud-translate==3.7.4",
+            "google-cloud-storage>=2.0.0",
+            "boto3>=1.21.21",
+            "boto3-stubs-lite[essential]>=1.21.21",
         ],
         # Individual sets of dependencies
         "pandas": ["pandas>=0.21.0"],
         "slack": ["aiohttp==3.7.4.post0", "slackclient==2.7.2"],
         "testing": ["responses==0.10.14"],
-        "translation": ["google-cloud-translate==2.0.1"],
+        "translation": ["google-cloud-translate==3.7.4"],
         "ssh": ["paramiko>=2.8.0"],
```

Comment:
We should also set individual sets for each of the clouds.  I think `s3` and `gcs` are good names:
```suggestion
        "ssh": ["paramiko>=2.8.0"],
        "gcs": ["google-cloud-storage>=2.0.0"],
        "s3": ["boto3>=1.21.21", "boto3-stubs-lite[essential]>=1.21.21"],
```